packages/orchestrator/lib/clients/validate.ts

```diff
@@ -178,9 +187,22 @@ export function validateTask(task: Task): Result<OrchestratorTask> {
             })
         );
     }
-    return Err(
-        `Cannot validate task ${JSON.stringify(task)}: ${stringifyError(sync.error || action.error || webhook.error || onEvent.error || syncAbort.error)}`
```

Comment:
returning the error is useless since if no parsing was successful it would always return the first sync.error